model/llm/llm.go

```diff
@@ -33,7 +33,7 @@ func NewLLMModel(provider provider.QueryProvider, modelIdentifier string) model.
 
 // llmGenerateTestForFilePrompt is the prompt used to query LLMs for test generation.
 var llmGenerateTestForFilePrompt = bytesutil.StringTrimIndentations(`
-	Given the following Go code file, provide a test file for this code.
+	Given the following Go code file "plain.test" with package "native", provide a test file for this code.
```

Comment:
Commit subject: "TMP"? Temporary? I usually prefix with TODO to make clear it is a TODO commit.